client/my-sites/customer-home/cards/education/quick-start-video/style.scss

```diff
@@ -0,0 +1,53 @@
+.quick-start-video .card {
+	@include breakpoint-deprecated( '>480px' ) {
+		padding: 16px 24px;
+	}
+}
+
+.customer-home__main .quick-start-video .card-heading {
+	margin-bottom: 4px;
+}
+
+.quick-start-video .inline-support-link {
+	margin-left: 4px;
+}
+
+.quick-start-video__all {
+	font-size: $font-body-small;
+}
+
+
+.quick-start-video__content {
```

Comment:
This is copying most of the styles from the educational content styles. You could get rid of this if you add the `educational-content` classes to the `QuickStartVideo` component:

```
<div class="quick-start-video__content educational-content">
  <div className="quick-start-video__content-wrapper educational-content__wrapper">
    // ...
  </div>
</div>
```